tests/console/check_system_info.pm

```diff
@@ -81,6 +81,20 @@ sub check_buildid {
     }
 }
 
+# SLE-21783: Update nodejs-common for SLE15 SP4
+# check the nodejs16 is releaseed to SLE15 SP3 & SLE15 SP4
+# check the nodejs-common points to nodejs16
+# test steps:
+# 1) install nodejs-common on SLES15SP3 and migrated SLES15SP4 when wsm is available in SCC_ADDONS
+# 2) check nodejs16 is installed together with nodejs-common
+sub check_nodejs_common {
+    record_info('SLE-21783', 'check nodejs-common');
+    zypper_call('in nodejs-common', log => 'zypper.log');
+    die "nodejs-common doesn't point to nodejs16 as default" unless (script_run('grep -E "nodejs16" /tmp/zypper.log') == 0);
+    script_run('rpm -qa | grep nodejs16');
+    zypper_call("rm nodejs-common", exitcode => [0]);
```

Comment:
The default exitcode should be 0, so no need write it. Just   zypper_call('rm nodejs-common'); is ok.

Reply:
> The default exitcode should be 0, so no need write it. Just zypper_call('rm nodejs-common'); is ok.

Done, thanks.